content/docs/command-reference/remote/add.md

```diff
@@ -159,16 +159,17 @@ For more information about the variables DVC supports, please visit
 ### Click for Microsoft Azure Blob Storage
 
 ```dvc
-$ dvc remote add myremote azure://my-container-name/path
-$ dvc remote modify --local myremote connection_string "my-connection-string"
+$ dvc remote add --local myremote azure://my-container-name/path
+$ dvc remote modify --local myremote connection_string \
+                            "my-connection-string"
 ```
```

Comment:
Did this per #1258 but should we start the example with the env var method, which doesn't need --local instead?

Reply:
it's fine I think, let's keep it this way for now